src/frontend/js/components/Search/index.tsx

```diff
@@ -172,4 +192,41 @@ const Search = ({ context }: CommonDataProps) => {
   );
 };
 
+interface FiltersPaneCloseButtonProps {
+  expanded: boolean;
+  controls: string;
+  onClick: () => void;
+  type?: 'top' | 'bottom';
+}
+
+const FiltersPaneCloseButton = ({
+  expanded,
+  controls,
+  onClick,
+  type = 'top',
+}: FiltersPaneCloseButtonProps) => (
+  <button
+    aria-expanded={expanded}
+    aria-controls={controls}
+    className={`search__filters__toggle search__filters__toggle--${type}`}
+    onClick={onClick}
+  >
+    {expanded ? (
+      <Fragment>
+        <Icon name="icon-cross" className="search__filters__toggle__icon" />{' '}
+        <span className="offscreen">
+          <FormattedMessage {...messages.hideFiltersPane} />
+        </span>
+      </Fragment>
+    ) : (
+      <Fragment>
+        <Icon name="icon-filter" className="search__filters__toggle__icon" />
+        <span className="offscreen">
+          <FormattedMessage {...messages.showFiltersPane} />
+        </span>
+      </Fragment>
+    )}
+  </button>
+);
+
```

Comment:
Should I create a new file for this button that is now a component by itself (since I use it twice now)?